app/services/dqt_record_check.rb

```diff
@@ -133,6 +133,12 @@ def magic_results
 
       # all matches 23 cohort start date - use 23/1/1900
       23 => CheckResult.new(magic_dqt_record(induction_start_date: Date.new(2023, 9, 1)), true, true, true, true, 4),
+
+      # all matches 23 cohort start date - use 24/1/1900
+      24 => CheckResult.new(magic_dqt_record(induction_start_date: Date.new(2024, 9, 1)), true, true, true, true, 4),
+
+      # all matches 25 cohort start date - use 25/1/1900
```

Comment:
might be worth adding this to the readme if we're going with this, or waiting till next year :)